packages/docregistry/style/index.css

```diff
@@ -8,14 +8,12 @@
   outline: none;
   overflow: hidden;
   margin-top: -1px;
```

Comment:
That's not the right way to solve that problem. The `border-bottom: none` should be set on the selected tab.